src/utils/wallet.js

```diff
@@ -731,6 +733,23 @@ const wallet = {
   decimalToInteger(value) {
     return Math.round(value*(10**hathorConstants.DECIMAL_PLACES));
   },
+
+  /**
+   * Returns a string map containing the identifiers for proposals currently being watched.
+   * @returns {Record<string,{ id:string, password:string }>}
+   */
+  getListenedProposalList() {
```

Comment:
Now I understand that it's not a list, it's a map. I think the name is misleading. Maybe `getListenedProposals` and `setListenedProposals` only?

Reply:
That's a good name: refactored on [77528b6](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/77528b6204b3b6be4963f9bb9fbc9e2678cef5d5).